pkgs/tools/graphics/zxing-cpp/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, pkgconfig, cmake, makeWrapper }:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  version = "20180320";
+  name = "zxing-cpp-${version}";
+
+  src = fetchFromGitHub {
+    owner = "glassechidna";
+    repo = "zxing-cpp";
+    rev = "5aad4744a3763d814df98a18886979893e638274";
+    sha256 = "07xshbh656a26bi8bii6ffw3fahnfj6755f6ybmmmplk55xxnlg8";
+  };
+
+  nativeBuildInputs = [ pkgconfig cmake makeWrapper ];
+  cmakeFlags = [ "-DCMAKE_BUILD_TYPE=Release" ];
```

Comment:
Apparently this is unnecessary: https://github.com/NixOS/nixpkgs/blob/e51f959fc769dd83c320d823f75ec306f7992497/pkgs/development/tools/build-managers/cmake/setup-hook.sh#L76